enhancements/ingress/set_eip_nlb_ingress.md

```diff
@@ -0,0 +1,983 @@
+      The kube-controller-manager logs may contain more details.)
+    reason: IngressControllerUnavailable
+    status: "False"
+    type: Available
+
+```
+
+3. Are there any pre-requisites for this EP ?
+  Yes, please refer section `Pre-requisites` under [Workflow Description](#Workflow Description)
+
+4. What happens when the cluster-admin specified invalid EIPs in install-config.yaml at installation time (day 0) ?
+
+   We would try to get the installer feasibly validate the `install-config.yaml` for day 0.
+   - For validation for BYOB subnets:
+  We will count if the number of cluster subnets provided by the user in the `install-config.yaml` were equal to 
+  the number of the eip allocations provided in the install-config.yaml. We will need to add validations 
+  in [pkg/asset/installconfig/aws/validation.go](https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/aws/validation.go#L186)
+  Discussion with the installer team to confirm this [approach](https://github.com/openshift/enhancements/pull/1593/files#r1617963511).
+
+  - For validation for  Installer Created Subnets:
+  The number of EIPs must match number of AZs in region. Installer team has confirmed with this [approach](https://github.com/openshift/enhancements/pull/1593/files#r1612258317).
+
+  - Validation for Invalid EIP which is generally when the EIP does not exist in the AWS environment or is not available.
+    For Example: is assigned to some instance or when count of EIPs does not match to the number of subnets.
+
+  Related comments:
+  - [discussion on separate the validation for BYOB subnets and installer-created subnets](https://github.com/openshift/enhancements/pull/1593#discussion_r1593385372)
+  - [workflow for when the installer creates the VPC and public subnets](https://github.com/openshift/enhancements/pull/1593#discussion_r1594164039)
+
+5. What happens when a cluster-admin adds invalid EIPs ?
+
+  The load balancer creation would not proceed and fail as per details mentioned in [subsection of Test Plan](#Test when Ingress Controller CR has been provided with invalid EIPAllocations.)
+
+6. What happens when a cluster-admin adds number of EIPs which are not equal to the number of public subnets for the VPC id tagged with cluster name.
+
+  The load balancer creation would not proceed and fail as per details mentioned in [subsection of Test Plan](#test-when-ingress-controller-cr-has-been-provided-with-number-of-eips-not-equal-to-the-number-of-public-subnets-for-vpc-tagged-with-your-cluster-name)
+
+7. What happens when a cluster-admin adds nothing under eipAllocations ?
+  
+  The load balancer would be created with no eipAllocations and the service annotation `service.beta.kubernetes.io/aws-load-balancer-eip-allocations` will be removed.
+  For more check [this](#Cluster admin wants operator to update the Load Balancer service for updated Ingress Controller eipAllocations)
+  The NLB will get the regular IP address assigned in that case.
```

Comment:
*nit* `regular` a bit of ambiguous term here, and also don't NLB almost always get multiple IPs assigned? Did you mean:

```suggestion
  AWS will randomly assign the NLB's IP addresses from the IP address pools available.
```